feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -58,45 +58,38 @@ public void start(Stage stage) {
             stage.setScene(scene);
             stage.show();
 
-
-            // create a new ArcGISMap with an oceans Basemap
+            // create a new ArcGISMap with the oceans basemap
             map = new ArcGISMap(Basemap.createOceans());
-            
-            // create a map view and set the ArcGISMap to it
+
+            // create portal
+            portal = new Portal("https://www.arcgis.com/");
+
+            // create a map view and set the map to it
             mapView = new MapView();
             mapView.setMap(map);
```

Comment:
I'd suggest swapping the order of these two lines. That way we deal with the map and map view first, and then create the portal. It should be a bit easier to follow.

```suggestion
            // create a map view and set the map to it
            mapView = new MapView();
            mapView.setMap(map);
            
            // create portal
            portal = new Portal("https://www.arcgis.com/");
```